### DISCUSSION TITLE
Question regarding blazor hybrid web view settings in project templates

### DISCUSSION THREAD

**Discussion by Mortezamir81** [bitfoundation/bitplatform#4146] [Q&A] [answered]
Hello
Could you please let me know what does this code snippet do?
Thanks in advance

![Screenshot_2023-03-23-16-41-49-322_com android chrome](https://user-images.githubusercontent.com/43121952/227222829-51a3955d-5798-44ff-8bc3-02a2630ab22a.jpg)

**Comment by mhrastegari** [comment, marked as answer]
Hello @mirshekarm thank you for contacting us.
- The iOS and macOS ones are for removing that white background color that you see on application start up 
- The Windows ones are for disabling both zoom with mouse wheel and refresh with F5 key of keyboard which is buggy.
- The Android ones are for giving you more access on the Blazor webview features and make it more like Blazor WASM and Server.
